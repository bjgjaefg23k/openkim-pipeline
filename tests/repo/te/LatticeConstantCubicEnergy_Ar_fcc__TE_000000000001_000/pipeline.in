@PATH[LatticeConstantCubicEnergy__TD_000000000000_000]
Ar
fcc
@MODELNAME
