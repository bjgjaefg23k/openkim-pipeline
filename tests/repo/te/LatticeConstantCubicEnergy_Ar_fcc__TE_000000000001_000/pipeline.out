lattice_constant :  PR_044492852008_000
energy :            PR_189562759581_000
